app/services/cert/certs.rb

```diff
@@ -8,6 +8,10 @@ def self.ca_cert
       File.read(Setting[:ssl_ca_file])
     end
 
+    def self.consumer_ca_cert
```

Comment:
```suggestion
    def self.candlepin_client_ca_cert
    def self.client_ca_cert
    def self.subscription_man_client_ca_cert
```

or

```suggestion

```

Trying to get away from using 'consumer' given I think that just leads to a layer of explanation. I suppose tihs could also be confusing.